.github/workflows/health-api-library.check.yml

```
@@ -62,23 +57,13 @@ jobs:
     - name: Build pinning example app and run integration tests
       run: >
         xcodebuild clean test
-        -project HealthAPILibrary/GiniHealthAPILibraryPinningExample/GiniHealthAPILibraryPinningExample.xcodeproj
-        -scheme "GiniHealthAPILibraryPinningExampleTests"
+        -project HealthAPILibrary/GiniHealthAPILibraryExample/GiniHealthAPILibraryExample.xcodeproj
+        -scheme "GiniHealthAPILibraryTests"
         -destination "${{ matrix.destination }}"
         -skip-testing:GiniHealthAPILibraryPinningExampleTests/HealthAPILibraryPinningWrongCertificatesTests
```

```suggestion
        -skip-testing:GiniHealthAPILibraryExampleTests/HealthAPILibraryPinningWrongCertificatesTests
```

Please check what are you running here